language/analysis/README.md

```diff
@@ -30,12 +30,13 @@ mvn clean compile assembly:single
 ```
 
 We can then run the assembled JAR file with the `java` command. The variable $COMMAND takes
-three values `entities`, `sentiment` or `syntax`.
+six values `entities-text`, `entities-file`, `sentiment-text`, `sentiment-file`,
+`syntax-text`, or `syntax-file`.
```

Comment:
Oi. Perhaps would be cleaner to just detect a gcs uri? As in:

```java
if (command.equals("entities")) {
  if (text.startsWith("gs://")) {
    app.analyzeEntitiesFile(text);
  } else {
    app.analyzeEntitiesText(text);
  }
} // ... etc
```
inside `Analyze.java`